src/plugins/chart_expressions/expression_xy/common/expression_functions/y_config.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { i18n } from '@kbn/i18n';
+import type { ExpressionFunctionDefinition } from '@kbn/expressions-plugin/common';
+import { Y_CONFIG } from '../constants';
+import { YConfig, YConfigResult } from '../types';
+import { commonYConfigArgs } from './common_y_config_args';
+
+export const yConfigFunction: ExpressionFunctionDefinition<
```

Comment:
I'm wondering which meaning has this `yConfig` name of the argument and expression function for now. Before, we had it as the configuration of y-axes. For now, we have the other one with `yAxisConfig` name. So, WDYT, how should we name the current expression, or is this name still reflect its purpose?

Reply:
Agree, Rename to `decorations`. Also now we have `dataDecorationConfig` instead of `yConfig` expression and `referenceLineDecorationConfig` instead of `extendeYConfig` expression